test/hotspot/jtreg/compiler/valhalla/inlinetypes/TestArrays.java

```diff
@@ -138,7 +138,7 @@
  * @modules java.base/jdk.internal.value
  *          java.base/jdk.internal.vm.annotation
  * @requires (os.simpleArch == "x64" | os.simpleArch == "aarch64")
- * @run main compiler.valhalla.inlinetypes.TestArrays 6
+ * @run main/othervm -XX:+IgnoreUnrecognizedVMOptions -XX:VerifyIterativeGVN=000 compiler.valhalla.inlinetypes.TestArrays 6
```

Comment:
That looks reasonable to do.

If we are only seeing the failures in the test VMs, then an easier solution would be to directly add the flag to the framework:
```
 InlineTypes.getFramework()
                   .addScenarios(scenarios[Integer.parseInt(args[0])])
                   .addFlags("-XX:+IgnoreUnrecognizedVMOptions -XX:VerifyIterativeGVN=000")
                   .addHelperClasses(MyValue1.class, MyValue2.class, MyValue2Inline.class)
                   .start();
```